### PR TITLE
Replace `RadioButton` expanders with `ComboBox`es

### LINKED ISSUE #9566: Settings dialog: Not possible to switch radio buttons with the keyboard
### Windows Terminal version (or Windows build number)

Windows Terminal Preview Version: 1.7.572.0

### Other Software

_No response_

### Steps to reproduce

1. Open Terminal.
2. Open Settings by pressing CTRL+,.
3. Tab to the radio buttons for the start mode. This is set to "Default" by default, but has options such as "Maximized" etc.
4. Try to switch between the set of five radio buttons using the keyboard, up and down arrows, or left and right arrows.

### Expected Behavior

Focus should switch to the other radio buttons, and the selection/check mark moves with it.

### Actual Behavior

Nothing happens.

Using arrow keys is the standard way to interact with a set of radio buttons.

### LINKED ISSUE #12648: Change radio button options in settings to listbox options
# Description of the new feature/enhancement

Since the settings area changed over to the Windows 11 style, extra navigation is needed to the any settings that use radio buttons to display the options.

image

An example below is the theme setting.

image

If these radio options were changed into listbox options (similar to the language option) then the currently set value would be visible immediatly and could be changed easier without the need for extra navigation to expand the box.

### PR DESCRIPTION
**Pull request by zadjii-msft**
Replaces all the `RadioButton` expanders with `ComboBox`es, which can have the options inline, as opposed to in the expander content. For example, here's a single commit with the changes for a single one of these settings: https://github.com/microsoft/terminal/commit/745c77d03e402a35901bfc5516486e315b0f673f

### Checklist
* [x] Closes #12648
* [x] Actually closes #9566 as well (by just removing all radio buttons)
* [x] I work here
* [x] Tested manually
* [x] I'd love @carlos-zamora to have an a11y pass at this, just to see if it's egregious or not.

### Before, after:

image

### CODE REVIEW

File: src/cascadia/TerminalSettingsEditor/Appearances.xaml [hunk 1]
```diff
@@ -23,6 +23,12 @@
                           x:DataType="local:EnumEntry">
                 <RadioButton Content="{x:Bind EnumName, Mode=OneWay}" />
```
Comment (DHowett) [inline src/cascadia/TerminalSettingsEditor/Appearances.xaml:24]: is the RadioButton version still used?

File: src/cascadia/TerminalSettingsEditor/Appearances.xaml [hunk 2]
```diff
@@ -23,6 +23,12 @@
                           x:DataType="local:EnumEntry">
                 <RadioButton Content="{x:Bind EnumName, Mode=OneWay}" />
             </DataTemplate>
+            <DataTemplate x:Key="EnumComboBoxTemplate"
+                          x:DataType="local:EnumEntry">
+                <StackPanel Orientation="Horizontal">
+                    <TextBlock Text="{x:Bind EnumName, Mode=OneWay}" />
```
[the remark this answers is not in the document]
Comment (zadjii-msft) [reply src/cascadia/TerminalSettingsEditor/Appearances.xaml:29]: > make sure the screen reader can read these. 

Oh yea, it's actually WAY better. Remember the "arrows don't work in Radio Buttons" issue? Well, that's gone now too!

> also check for the availability of Typeahead Search (typing in a combo box)

Sure does 🙏

**Review by lhecker — APPROVED** [review]
Beautiful!

nit: Dustin's question regarding whether `EnumRadioButtonTemplate` is still needed.

**Comment by carlos-zamora** [comment]
> * [ ]  I'd love @carlos-zamora to have an a11y pass at this, just to see if it's egregious or not.

Looks good to me. So much better than before. Thank you!

File: src/cascadia/TerminalSettingsEditor/Appearances.xaml [hunk 3]
```diff
@@ -23,6 +23,12 @@
                           x:DataType="local:EnumEntry">
                 <RadioButton Content="{x:Bind EnumName, Mode=OneWay}" />
             </DataTemplate>
+            <DataTemplate x:Key="EnumComboBoxTemplate"
+                          x:DataType="local:EnumEntry">
+                <StackPanel Orientation="Horizontal">
```
Comment (carlos-zamora) [inline src/cascadia/TerminalSettingsEditor/Appearances.xaml:28]: Do you even need the `StackPanel`? It seems like it only has one child so it's not really doing anything.

File: src/cascadia/TerminalSettingsEditor/Appearances.xaml [hunk 4]
```diff
@@ -472,12 +478,12 @@
             <local:SettingContainer x:Uid="Appearance_IntenseTextStyle"
                                     ClearSettingValue="{x:Bind Appearance.ClearIntenseTextStyle}"
                                     HasSettingValue="{x:Bind Appearance.HasIntenseTextStyle, Mode=OneWay}"
-                                    SettingOverrideSource="{x:Bind Appearance.IntenseTextStyleOverrideSource, Mode=OneWay}"
-                                    Style="{StaticResource ExpanderSettingContainerStyle}">
+                                    SettingOverrideSource="{x:Bind Appearance.IntenseTextStyleOverrideSource, Mode=OneWay}">
                 <muxc:RadioButtons AutomationProperties.AccessibilityView="Content"
-                                   ItemTemplate="{StaticResource EnumRadioButtonTemplate}"
+                                   ItemTemplate="{StaticResource EnumComboBoxTemplate}"
```
Comment (carlos-zamora) [inline src/cascadia/TerminalSettingsEditor/Appearances.xaml:483]: You're using an `EnumComboBoxTemplate` in a `muxc:RadioButtons`? I think you missed this one haha.

File: src/cascadia/TerminalSettingsEditor/Appearances.xaml (hunk 3 above)
Comment (carlos-zamora) [reply src/cascadia/TerminalSettingsEditor/Appearances.xaml:28]: This probably applies to the copied over version in the other files.

File: src/cascadia/TerminalSettingsEditor/Appearances.xaml (hunk 1 above)
Comment (carlos-zamora) [reply src/cascadia/TerminalSettingsEditor/Appearances.xaml:24]: This probably applies to the copied over version in the other files.

File: src/cascadia/TerminalSettingsEditor/Profiles_Advanced.xaml [hunk 5]
```diff
@@ -94,20 +100,19 @@
...
+                              Style="{StaticResource ComboBoxSettingStyle}" />
                 </local:SettingContainer>
 
                 <!--  Bell Style  -->
                 <local:SettingContainer x:Uid="Profile_BellStyle"
                                         ClearSettingValue="{x:Bind Profile.ClearBellStyle}"
                                         HasSettingValue="{x:Bind Profile.HasBellStyle, Mode=OneWay}"
-                                        SettingOverrideSource="{x:Bind Profile.BellStyleOverrideSource, Mode=OneWay}"
-                                        Style="{StaticResource ExpanderSettingContainerStyle}">
+                                        SettingOverrideSource="{x:Bind Profile.BellStyleOverrideSource, Mode=OneWay}">
```
Comment (carlos-zamora) [inline src/cascadia/TerminalSettingsEditor/Profiles_Advanced.xaml:115]: This doesn't seem right.

**Review by carlos-zamora — CHANGES_REQUESTED** [review]

File: src/cascadia/TerminalSettingsEditor/Profiles_Advanced.xaml (hunk 5 above)
Comment (zadjii-msft) [reply src/cascadia/TerminalSettingsEditor/Profiles_Advanced.xaml:115]: wait what doesn't seem right?

Comment (carlos-zamora) [reply src/cascadia/TerminalSettingsEditor/Profiles_Advanced.xaml:115]: Removing the `Style`. Really even modifying this SettingContainer at all. This doesn't have any RadioButtons in it.

Comment (zadjii-msft) [reply src/cascadia/TerminalSettingsEditor/Profiles_Advanced.xaml]: 🤦

**Review by carlos-zamora — APPROVED** [review]
Looks great! Thanks!

**Pull request merged**